.code-samples.meilisearch.yaml

```diff
@@ -467,16 +467,19 @@ getting_started_add_documents_md: |-
   curl \
     -X POST 'http://localhost:7700/indexes/movies/documents?primaryKey=id' \
     -H 'Content-Type: application/json' \
+    -H 'Authorization: Bearer aSampleMasterKey' \
```

Comment:
```suggestion
    -H 'Authorization: Bearer MASTER_KEY' \
```
We use the `MASTER_KEY ` format [everywhere else in the docs](https://www.meilisearch.com/docs/reference/api/overview#:~:text=Placeholder%20text%20is%20in%20uppercase%20characters%20with%20underscore%20delimiters%2C%20for%20example%2C%20MASTER_KEY)

Reply:
The problem with it is that someone who copypastes `MASTER_KEY` will get a warning about the size of the master key and will be working on an unsecured instance (and getting errors if they continue to copypaste other samples) for the remainder of the tutorial.

Reply:
True, but `aSampleMasterKey` or `MASTER_KEY` is a placeholder 🤔

Reply:
Well, the way I see it `MASTER_KEY` is a placeholder. `aSampleMasterKey` is just a weak master key.

In my head, the quick start plays according to different rules than, say, the API reference. I want users to copypaste every code sample here without changing anything and still see Meilisearch working. Like, less teaching, more demonstrating.

Reply:
That makes sense! Should we add a note or something to clarify why we're using two different "formats" in the docs?

Reply:
Maybe [MASTER_KEY] indicates that the value should be changed?

Reply:
Unfortunately, I don't remember the documentation website I've seen, but what we could do is to ask the user what is their master key, and then apply it to every page where we need one.

Something like this done by pipefy https://developers.pipefy.com/reference/authentication:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/4116980/236318929-aa002fc6-0512-486e-a378-8b6d60ac3252.png">

Reply:
> Maybe [MASTER_KEY] between brackets indicates that the value should be changed?

It's been a while now, but we did some research in the past and I vaguely recall some sources recommending against using brackets. We opted for MASTER_KEY, indicating the placeholder should be replaced whenever feasible.

---

> ask the user what is their master key, and then apply it to every page where we need one

This is an interesting idea, but out of scope for this PR. I created an issue where we can properly discuss this.

Reply:
> That makes sense! Should we add a note or something to clarify why we're using two different "formats" in the docs?

I was reading about placeholders just now and one suggestion I liked was to indicate the placeholder text should be replaced below the code sample. Something like 

```markdown
[code sample block]

Replace MASTER_KEY with your instance's master key.
```

I will indicate this clearly in the text here, so users are aware of the context, but we might want to discuss this more in an issue.